### PR TITLE
Deprecate d2 scripts

### PR DESCRIPTION
**Pull request by YuvHayun**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Description
After consulting @bakatzir, we decided to deprecate d2 python 2 scripts rather than translate them to python3.

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/D2/Scripts/Autoruns/Autoruns.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+args:
+- name: system
+  required: true
+  default: true
+  description: Name of the system from which to collect data
+scripttarget: 0
+dependson: {}
+timeout: 0s
+fromversion: 5.0.0
+dockerimage: demisto/python:2.7.18.24398
```
Comment (tkatzir) [inline Packs/D2/Scripts/Autoruns/Autoruns.yml:23]: Python 2.7?

File: Packs/D2/Scripts/Autoruns/Autoruns.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+commonfields:
+  id: Autoruns
+  version: -1
+name: Autoruns
+script: ''
+type: python
+subtype: python2
```
Comment (tkatzir) [inline Packs/D2/Scripts/Autoruns/Autoruns.yml:7]: python2?

File: Packs/D2/Scripts/D2Remove/D2Remove.yml [hunk 3]
```diff
@@ -6,7 +6,8 @@ script: ''
 type: python
 subtype: python2
```
Comment (tkatzir) [inline Packs/D2/Scripts/D2Remove/D2Remove.yml:7]: python2?

File: Packs/D2/Scripts/O365SearchEmails/O365SearchEmails.yml [hunk 4]
```diff
@@ -0,0 +1,40 @@
+commonfields:
+  id: O365SearchEmails
+  version: -1
+name: O365SearchEmails
+script: ''
+type: python
+subtype: python2
```
Comment (tkatzir) [inline Packs/D2/Scripts/O365SearchEmails/O365SearchEmails.yml:7]: python2?

File: Packs/D2/Scripts/O365SearchEmails/O365SearchEmails.yml [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+- name: system
+  required: true
+  description: Windows system to run the powershell script
+- name: delete
+  description: if set to "true" (case insensitive) - the script will also delete found emails. Default is false.
+scripttarget: 0
+dependson: {}
+timeout: 0s
+fromversion: 5.0.0
+dockerimage: demisto/python:2.7.18.24398
```
Comment (tkatzir) [inline Packs/D2/Scripts/O365SearchEmails/O365SearchEmails.yml:38]: python2?

File: Packs/D2/Scripts/RegCollectValues/RegCollectValues.yml [hunk 6]
```diff
@@ -0,0 +1,25 @@
+commonfields:
+  id: RegCollectValues
+  version: -1
+name: RegCollectValues
+script: ''
+type: python
+subtype: python2
```
Comment (tkatzir) [inline Packs/D2/Scripts/RegCollectValues/RegCollectValues.yml:7]: python2?

File: Packs/D2/Scripts/RegCollectValues/RegCollectValues.yml [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+args:
+- name: regpath
+  required: true
+  default: true
+  description: The registry path of the key to query
+scripttarget: 0
+dependson: {}
+timeout: 0s
+fromversion: 5.0.0
+dockerimage: demisto/python:2.7.18.24398
```
Comment (tkatzir) [inline Packs/D2/Scripts/RegCollectValues/RegCollectValues.yml:23]: python2?

**Review by tkatzir — COMMENTED** [review]
python2?

**Comment by YuvHayun** [comment]
@tkatzir yes, they're all python2.. I needed to split the files because the previous name was problematic (script-<script name) but I didn't change the py version.

**Review by JudahSchwartz — APPROVED** [review]
approving since leaving as py2 is fine as theyre all being deprecated

**Comment by YuvHayun** [comment]
This pr needs to be force merged since the scripts were split and the old files were deleted.

**Pull request merged**